apis/externalsecrets/v1beta1/externalsecret_types.go

```diff
@@ -212,17 +212,31 @@ const (
 	ExternalSecretDecodeNone      ExternalSecretDecodingStrategy = "None"
 )
 
-// +kubebuilder:validation:MinProperties=1
-// +kubebuilder:validation:MaxProperties=1
 type ExternalSecretDataFromRemoteRef struct {
 	// Used to extract multiple key/value pairs from one secret
 	// +optional
 	Extract *ExternalSecretDataRemoteRef `json:"extract,omitempty"`
 	// Used to find secrets based on tags or regular expressions
 	// +optional
 	Find *ExternalSecretFind `json:"find,omitempty"`
+
+	// Used to rewrite secret Keys after getting them from the secret Provider
+	// +optional
+	Rewrite []ExternalSecretRewrite `json:"rewrite,omitempty"`
 }
 
+type ExternalSecretRewrite struct {
+	// Rewrite using regular expressions
```

Comment:
Could you please expand this a bit? This is very helpful when using `kubectl explain` or the reference docs.

Reply:
I did some suggestions there, but I'm not sure how more/less verbose I should be. WDYT of the current version?

Reply:
This is good for now!